### ISSUE TITLE
CMS [COGNITION]: Headings MUST look like headings, links MUST look like links

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8252]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Until the search result headings are linked to detail pages the headings should be styled only as headings, without the link blue. Once they are links, they should be styled with the class `.vads-u-color--link-default` as they currently are, and also have an underline.

### Current 
This heading is not linked, but looks like a link (minus an underline).
![Screen Shot 2020-04-21 at 8 07 14 PM](https://user-images.githubusercontent.com/57469/79926757-b9818000-840b-11ea-9ba2-e2057f8a9032.png)

### Should
This is how an unlinked heading **should** look in this usage.
![Screen Shot 2020-04-21 at 8 05 39 PM](https://user-images.githubusercontent.com/57469/79926770-c2725180-840b-11ea-8db8-38f178ca5a9e.png)

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a user I want to view the page content and know which content is interactive.

## Environment

* Operating System: all
* Browser: all
* Server destination: production

## Steps to Recreate

1. Enter https://www.va.gov/find-forms?q=caregiver in browser
2. Verify that the unlinked heading for the search result is `#004795` rather than the `#323a45` for unlinked content.

## Possible Fixes (optional)

The search result heading currently has a class `.vads-u-color--link-default` applied, but the headings are not links. Consider removing this class since these are not links.

## WCAG or Vendor Guidance (optional)

* [WebAIM Links and hypertext](https://webaim.org/techniques/hypertext/link_text)
* [WCAG 2.0 - Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/TR/WCAG20-TECHS/F73.html)
* [HTML5 Accessibility: aria-hidden and role=”presentation”](http://john.foliot.ca/aria-hidden/)

**Comment by 1Copenut** [comment]
@jenstrickland Can you provide a WCAG success criterion for this one? I had this flagged as one of my notes to review also. I agree that it should be black text instead.

I feel it's more of a `2` level issue given the heading looks like a link because of color.

**Comment by stephorkandatx** [comment]
@jenstrickland I believe we have resolved the noted issue within another ticket:  #8135 

Before we close this out, would you mind taking a look? The change can be seen at va.gov/find-forms (with the credentials) & the screenshot below

Screenshot-
![image](https://user-images.githubusercontent.com/63361321/80640065-869d4480-8a28-11ea-8df9-dd8fc8065df7.png)

**Comment by stephorkandatx** [comment]
@jenstrickland apologies for the notifications.  This fix is not yet ready. We will make the correction and I will let you know when it's ready

**Comment by jenstrickland** [comment]
Fix validated.
